gc/base/MemorySpace.hpp

```diff
@@ -155,7 +155,7 @@ class MM_MemorySpace : public MM_BaseVirtual
 	MMINLINE MM_MemorySubSpace *getMemorySubSpaceList() { return _memorySubSpaceList; }
 
 	void systemGarbageCollect(MM_EnvironmentBase *env, uint32_t gcCode);
-	void localGarbageCollect(MM_EnvironmentBase *env);
```

Comment:
Need default value due to cross dependent changes - default value will be removed once both OMR and OpenJ9 are merged.